docs/example.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: app.k8s.io/v1alpha1
+apiVersion: app.k8s.io/v1beta1
```

Comment:
should the `app.k8s.io` prefix be there? It was removed here https://github.com/kubernetes-sigs/application/pull/52/commits/18cbb53830259d38a1f7bbe77142ba7810b9f09a#diff-04c6e90faac2675aa89e2176d2eec7d8R183